src/js/components/FormField/FormField.js

```diff
@@ -221,13 +221,26 @@ const FormField = forwardRef(
     contents = <Box {...contentProps}>{contents}</Box>;
 
     let borderColor;
-    if (focus && !normalizedError) {
+
+    if (disabled) {
+      borderColor =
+        formField.disabled.border && formField.disabled.border.color;
+    } else if (focus && !normalizedError) {
       borderColor = 'focus';
     } else if (normalizedError) {
       borderColor = (border && border.error.color) || 'status-critical';
     } else {
       borderColor = (border && border.color) || 'border';
     }
+
+    const labelStyle = formField.label;
+
+    if (disabled) {
+      labelStyle.color = formField.disabled.label
```

Comment:
```suggestion
      labelStyle.color = formField.disabled && formField.disabled.label
```
Added this precheck in case the user decides to override the formField object unless you think we can live without it.